migration-roadmap/progress-circle.md

```diff
@@ -0,0 +1,242 @@
+-           </div>
+-       </div>
+-   </div>
++   <svg fill="none" width="100%" height="100%" class="spectrum-outerCircle">
++       <circle class="spectrum-innerCircle" cx="50%" cy="50%" r="calc(50% - 3px)" stroke-width="2" />
++       <circle
++           cx="50%"
++           cy="50%"
++           class="spectrum-ProgressCircle-track"
++           r="calc(50% - 1.5px)"
++       />
++       <circle
++           cx="50%"
++           cy="50%"
++           r="calc(50% - 1.5px)"
++           class="spectrum-ProgressCircle-fill"
++           pathLength="100"
++           stroke-dasharray="100 200"
++           stroke-dashoffset="50"
++           stroke-linecap="round"
++       />
++   </svg>
+</div>
+```
+
+</details>
+
+### CSS => SWC mapping
+
+| CSS selector                                                                                  | Attribute or slot          | Status           |
+| --------------------------------------------------------------------------------------------- | -------------------------- | ---------------- |
+| `.spectrum-ProgressCircle--sizeS`                                                             | `size="s"`                 | Implemented      |
+| `.spectrum-ProgressCircle--sizeL`                                                             | `size="l"`                 | Implemented      |
+| `.spectrum-ProgressCircle--indeterminate`                                                     | `indeterminate`            | Implemented      |
+| `.spectrum-ProgressCircle.spectrum-ProgressCircle--staticBlack`                               | `static-color="black"`     | Missing from WC  |
+| `.spectrum-ProgressCircle.spectrum-ProgressCircle--staticWhite`                               | `static-color="white"`     | Implemented      |
+| `.spectrum-ProgressCircle`                                                                    | Base component             | Implemented      |
+| `.spectrum-ProgressCircle-fill`                                                               | Internal fill element      | Implemented      |
+| `.spectrum-ProgressCircle-track`                                                              | Internal track element     | Implemented      |
+| `.spectrum-ProgressCircle--indeterminate .spectrum-ProgressCircle-fill`                       | Indeterminate fill styling | Implemented      |
+| `.spectrum-ProgressCircle:not(.spectrum-ProgressCircle--indeterminate) .spectrum-innerCircle` | Determinate inner circle   | Missing from CSS |
+| `.spectrum-ProgressCircle:not(.spectrum-ProgressCircle--indeterminate) .spectrum-outerCircle` | Determinate outer circle   | Missing from CSS |
+|                                                                                               | `label`                    | Missing from CSS |
+|                                                                                               | `progress`                 | Missing from CSS |
```

Comment:
Is `percentFilled` the `progress` equivalent in CSS? Maybe it's not technically missing, but we call it something else.